### ISSUE TITLE
Requiring academic or institutional email address from references

### ISSUE THREAD

**Issue by lamawmouk** [MIT-LCP/physionet-build#1797]
I am looking to require users to provide reference emails with an academic or institutional email to ease the flow of the credentialing pipeline. I chatted with Tom and Benjamin and they agree about the change. We would have a list of known domains and continue to add to that list afterwards. Any thoughts on this change not being implemented?

**Comment by alistairewj** [comment]
Can you give more detail on what specifically you have in mind? Do you want to disallow submitting an application if the reference e-mail address is not from a specific domain, i.e. some kind of input form validation?

**Comment by lamawmouk** [comment]
Yes and if the email address is a third-party email like gmail.com, yahoo.com, 163.com, qq.com, outlook.com etc.

**Comment by tompollard** [comment]
Overall I support the idea that applicants should provide either (1) an institutional email address or (2) a reference with an institutional email address. I agree the validation would probably have to be done by excluding known non-institutional addresses (e.g. hotmail, gmail, etc).

@alistairewj I think @lamawmouk's thought would be that this would be tied into the traffic light system described at: https://github.com/MIT-LCP/physionet-build/issues/1712. e.g if a user could not provide an institutional email address, the traffic light would be red (meaning the response is likely to be reject, asking for more information).

It would be good to know what proportion of people would not be able to meet this requirement. How would we deal with cases where the user does not have an institutional email address? Is there something that we could accept as an alternative?
